benches/benches/bevy_ecs/commands.rs

```diff
@@ -0,0 +1,101 @@
+use bevy::ecs::{
+    system::{CommandQueue, Commands},
+    world::World,
+};
+use criterion::{criterion_group, criterion_main, Criterion};
+use rand::{rngs::StdRng, RngCore, SeedableRng};
+
+criterion_group!(benches, empty_cmds, cmds, rng_cmds);
+criterion_main!(benches);
+
+struct A;
+struct B;
+struct C;
+
+fn empty_cmds(criterion: &mut Criterion) {
+    let mut group = criterion.benchmark_group("empty_commands");
+    group.warm_up_time(std::time::Duration::from_millis(500));
+    group.measurement_time(std::time::Duration::from_secs(5));
+
+    group.bench_function("0_entities", |bencher| {
+        let mut world = World::default();
+        let mut command_queue = CommandQueue::default();
+
+        bencher.iter(|| {
+            command_queue.apply(&mut world);
+        });
+    });
+
+    group.finish();
+}
+
+fn cmds(criterion: &mut Criterion) {
+    let mut group = criterion.benchmark_group("commands");
+    group.warm_up_time(std::time::Duration::from_millis(500));
+    group.measurement_time(std::time::Duration::from_secs(5));
+
+    for i in (1..5).map(|i| i * 2) {
+        let entity_count = i * 1000;
+
+        group.bench_function(format!("{}_entities", entity_count), |bencher| {
+            let mut world = World::default();
+            let mut command_queue = CommandQueue::default();
+
+            bencher.iter(|| {
+                let mut commands = Commands::new(&mut command_queue, &world);
+                for _ in 0..entity_count {
+                    commands.spawn().insert(A).insert(B).insert(C).despawn();
+                }
+                drop(commands);
+                command_queue.apply(&mut world);
+            });
+        });
+    }
+
+    group.finish();
+}
+
+fn rng_cmds(criterion: &mut Criterion) {
+    let mut group = criterion.benchmark_group("rng_commands");
+    group.warm_up_time(std::time::Duration::from_millis(500));
+    group.measurement_time(std::time::Duration::from_secs(5));
+
+    for i in (1..5).map(|i| i * 2) {
+        let entity_count = i * 1000;
+
+        group.bench_function(format!("{}_entities", entity_count), |bencher| {
+            let mut world = World::default();
+            let mut command_queue = CommandQueue::default();
+
+            let mut rng = StdRng::seed_from_u64(42);
```

Comment:
We use see if using SmallRng is more performant to minimize the impact of the randomizer in benchmark.